samples/notebooks/dotnet/09-BingSearch-using-kernel-prince.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BingSearch using Kernel \n",
+    "\n",
+    "This notebook explains how to integragte BingSearch feature in your Semantic Kernel  to get latest information from the internet.\n",
+    "\n",
+    "To using BingSeach you simply need *Bing Search Api Key* . You can get the api by creating [Bing Search resource](https://learn.microsoft.com/en-us/bing/search-apis/bing-web-search/create-bing-search-service-resource) in Azure . \n",
+    "\n",
+    "In free Instance you will get *1000* transactions per month and only 3 Transactions per second valid .\n",
+    "\n",
+    "Check other pricing from [here](https://www.microsoft.com/en-us/bing/apis/pricing).\n",
```

Comment:
These lines are unnecessary, please remove. Users will be prompted with pricing information when they create their API key.